.eslintrc.js

```diff
@@ -1,7 +1,12 @@
 module.exports = {
 	root: true,
+	env: {
+		es6: true,
+		node: true,
+	},
 	extends: [
-		'kentcdodds',
```

Comment:
`eslint-config-kentcdodds` uses some outdated packages which block us from upgrading TypeScript dependencies properly. It didn't bring many rules so decided just use `eslint:recommended`, `plugin:import/recommended` and configure some rules manually.

Reply:
I like this change! We should apply it anyway.